migration/migrating-3-4/troubleshooting.adoc

```diff
@@ -8,7 +8,15 @@ toc::[]
 
 You can view the migration Custom Resources (CRs) and download logs to troubleshoot a failed migration.
 
+You can manually roll back a failed migration.
+
+[NOTE]
+====
+Manual rollback is only required if an application has been quiesced, to prevent data corruption. This procedure is not required if the application was not quiesced, because the original application is still running on the source cluster.
```

Comment:
Can you s/quiesced/paused and not lose any meaning? If so, I'd do it in all places in the PR. (It looks like you're removing the only place it's currently used.)

I'd move "to prevent data corruption" to the start of the sentence.

Reply:
I'm thinking that "stopped" might be a better word. I just did a search of the openshift docs and "stopped" seems to be the preferred word for pods (rather than pausing, halting, or quiescing). The UI itself isn't very helpful because the check box says "halt transactions". In a way, that's true, but the transactions between the pods and storage are halted by stopping the pods, so it would be accurate to say that the application is stopped. 

`I'd move "to prevent data corruption" to the start of the sentence.`  To the start of which sentence? The first sentence in the note or the sentence preceding the note? I think it might make sense to move it to the sentence preceding the note.

Reply:
Then please use "stopped" and consider filing a UI bug to make that check box text more helpful. :)

I think that moving it to the start of the note or the sentence before the note would work. The meaning's slightly different depending on where you put it, so please pick the one that is best.

Reply:
You're right. That check box has been bothering me for ages and I keep putting off creating that bug. Will do!